Packs/AzureDevOps/Integrations/AzureDevOps/AzureDevOps.yml

```diff
@@ -773,10 +773,2220 @@ script:
+      contextPath: AzureDevOps.PullRequestThread.comments._links.repository.href
+      description: Link to the repository where the comments are.
+    - type: String
+      contextPath: AzureDevOps.PullRequestThread.comments._links.threads.href
+      description: Link to the threads.
+    - type: String
+      contextPath: AzureDevOps.PullRequestThread.comments._links.pullRequests.href
+      description: Link to the pull request.
+    - type: Unknown
+      contextPath: AzureDevOps.PullRequestThread.threadContext
+      description: Extended context information unique to pull requests.
+    - type: String
+      contextPath: AzureDevOps.PullRequestThread.properties.CodeReviewThreadType.$type
+      description: The type of the code review thread.
+    - type: String
+      contextPath: AzureDevOps.PullRequestThread.properties.CodeReviewThreadType.$value
+      description: The content in the code review thread.
+    - type: String
+      contextPath: AzureDevOps.PullRequestThread.properties.CodeReviewReviewersUpdatedNumAdded.$type
+      description: A number (Int32).
+    - type: Number
+      contextPath: AzureDevOps.PullRequestThread.properties.CodeReviewReviewersUpdatedNumAdded.$value
+      description: Number of code reviewers updated the pull request.
+    - type: String
+      contextPath: AzureDevOps.PullRequestThread.properties.CodeReviewReviewersUpdatedNumChanged.$type
+      description: A number (Int32).
+    - type: Number
+      contextPath: AzureDevOps.PullRequestThread.properties.CodeReviewReviewersUpdatedNumChanged.$value
+      description: Number of code reviewers changed the pull request.
+    - type: String
+      contextPath: AzureDevOps.PullRequestThread.properties.CodeReviewReviewersUpdatedNumDeclined.$type
+      description: A number (Int32).
+    - type: Number
+      contextPath: AzureDevOps.PullRequestThread.properties.CodeReviewReviewersUpdatedNumDeclined.$value
+      description: Number of code reviewers declined the pull request.
+    - type: String
+      contextPath: AzureDevOps.PullRequestThread.properties.CodeReviewReviewersUpdatedNumRemoved.$type
+      description: A number (Int32).
+    - type: Number
+      contextPath: AzureDevOps.PullRequestThread.properties.CodeReviewReviewersUpdatedNumRemoved.$value
+      description: Number of code reviewers are removed.
+    - type: String
+      contextPath: AzureDevOps.PullRequestThread.properties.CodeReviewReviewersUpdatedAddedIdentity.$type
+      description: A number (Int32).
+    - type: String
+      contextPath: AzureDevOps.PullRequestThread.properties.CodeReviewReviewersUpdatedAddedIdentity.$value
+      description: Number of code reviewers added identity.
+    - type: String
+      contextPath: AzureDevOps.PullRequestThread.properties.CodeReviewReviewersUpdatedByIdentity.$type
+      description: A number (Int32).
+    - type: String
+      contextPath: AzureDevOps.PullRequestThread.properties.CodeReviewReviewersUpdatedByIdentity.$value
```

Comment:
- Are these paths correct?
- The descriptions that are "A number (Int32) need to be replaced with a good description.

Reply:
They are correct and this is actually the type - A number (Int32)